pandas/tests/internals/test_internals.py

```diff
@@ -297,7 +297,10 @@ def test_delete(self):
         assert (newb.values[1] == 1).all()
 
         newb = self.fblock.copy()
-        with pytest.raises(Exception):
+
+        msg = "index 3 is out of bounds for axis 0 with size 3"
+
+        with pytest.raises(Exception, match=msg):
```

Comment:
when you come across Exception can you either change the type to the Exception type or add a comment # raises type Exception to help track these

Reply:
Oh, didn't notice.

I will know for the next time